apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -0,0 +1,363 @@
+import MaterialTable, { Column, OrderByCollection } from '@material-table/core';
+import { t, TFunction } from 'i18next';
+import React, { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { ProposalsFilter } from 'generated/sdk';
+import { useCallsData } from 'hooks/call/useCallsData';
+import { useProposalStatusesData } from 'hooks/settings/useProposalStatusesData';
+import { useTechniquesData } from 'hooks/technique/useTechniquesData';
+import { StyledContainer, StyledPaper } from 'styles/StyledComponents';
+import {
+  addColumns,
+  fromArrayToCommaSeparated,
+  setSortDirectionOnSortField,
+} from 'utils/helperFunctions';
+import { tableIcons } from 'utils/materialIcons';
+
+import { ProposalViewData, useProposalsCoreData } from './useProposalsCoreData';
+import { useXpressInstrumentsData } from './useXpressInstrumentsData';
+import XpressProposalFilterBar from './XpressProposalFilterBar';
+
+const XpressProposalTable = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [selectedProposals, setSelectedProposals] = useState<
+    ProposalViewData[]
+  >([]);
+  const { techniques, loadingTechniques } = useTechniquesData();
+  const { proposalStatuses, loadingProposalStatuses } =
+    useProposalStatusesData();
+  const [currentPage, setCurrentPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(20);
+  const PREFETCH_SIZE = 200;
+
+  const [searchParams, setSearchParams] = useSearchParams({});
+
+  const callId = searchParams.get('callId');
+  const instrument = searchParams.get('instrument');
+  const technique = searchParams.get('technique');
+  const proposalId = searchParams.get('proposalId');
+  const to = searchParams.get('to');
+  const from = searchParams.get('from');
+  const proposalStatusId = searchParams.get('proposalStatusId');
+  const search = searchParams.get('search');
+  const selection = searchParams.getAll('selection');
+  const sortField = searchParams.get('sortField');
+  const sortDirection = searchParams.get('sortDirection');
+
+  type QueryParameters = {
+    query: {
+      first?: number;
+      offset?: number;
+    };
+    searchText?: string | undefined;
+  };
+  const [queryParameters, setQueryParameters] = useState<QueryParameters>({
+    query: {
+      first: PREFETCH_SIZE,
+      offset: 0,
+    },
+    searchText: search ?? undefined,
+  });
+
+  const [proposalFilter, setProposalFilter] = useState<ProposalsFilter>({
+    callId: callId ? +callId : undefined,
+    instrumentFilter: {
+      instrumentId: instrument ? +instrument : null,
+      showAllProposals: !instrument,
```

Comment:
If l select a technique with multiple instrument will this filter all or just one.

Reply:
Hope I understood the question correctly. If a technique alone is selected in technique filter, it will check for proposal which has that particular technique attached, independent of instruments in technique. 
If any instrument is selected from instrument filter, it will now check for the technique to which that particular instrument is attached and return all proposal which has that particular technique.